lib/foreman_remote_execution_core/fake_script_runner.rb

```diff
@@ -0,0 +1,82 @@
+module ForemanRemoteExecutionCore
+  class ScriptRunner < ForemanTasksCore::Runner::Base
```

Comment:
I guess we hide this behind `ForemanRemoteExecutionCore.runner_class`, that could be memoized to avoid the checking every time.